pkgs/by-name/cy/cyrus-imapd/package.nix

```diff
@@ -0,0 +1,192 @@
+{
+  autoreconfHook,
+  bison,
+  brotli,
+  coreutils,
+  cunit,
+  cyrus-imapd,
+  cyrus_sasl,
+  fetchFromGitHub,
+  fig2dev,
+  flex,
+  icu,
+  jansson,
+  lib,
+  libbsd,
+  libcap,
+  libchardet,
+  libical,
+  libmysqlclient,
+  libsrs2,
+  libuuid,
+  libxml2,
+  makeWrapper,
+  nghttp2,
+  openssl,
+  pcre2,
+  perl,
+  pkg-config,
+  postgresql,
+  rsync,
+  shapelib,
+  sqlite,
+  stdenv,
+  testers,
+  unixtools,
+  valgrind,
+  withCalalarmd ? true,
+  withHttp ? true,
+  withJMAP ? true,
+  withMySQL ? false,
+  withNNTP ? false,
+  withPgSQL ? false,
+  withSQLite ? true,
+  withXapian ? true,
+  wslay,
+  xapian,
+  zlib,
+}:
+stdenv.mkDerivation rec {
```

Comment:
I think the `finalAttrs` pattern should be used here to allow for more effective overriding, see https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes